packages/unlock-js/src/unlockService.js

```diff
@@ -21,6 +21,10 @@ export default class UnlockService {
     if (!this.networks[networkId]) {
       throw new Error(`Missing config for ${networkId}`)
     }
+    // for convenience, pass directly an ethers provider in the `networks` contructor
+    if (this.networks[networkId].ethersProvider) {
+      return this.networks[networkId].ethersProvider
+    }
```

Comment:
this is a require change in the library itself, so the only blocker I can think of.